devlib/target.py

```diff
@@ -167,13 +167,17 @@ def number_of_cpus(self):
     @property
     @memoized
     def number_of_nodes(self):
-        num_nodes = 0
-        nodere = re.compile(r'^\./node\d+\s*$')
-        output = self.execute('cd /sys/devices/system/node && busybox find . -maxdepth 1', as_root=self.is_rooted)
-        for entry in output.splitlines():
-            if nodere.match(entry):
-                num_nodes += 1
-        return num_nodes
+        try:
+            output = self.execute('cd /sys/devices/system/node && busybox find . -maxdepth 1', as_root=self.is_rooted)
```

Comment:
`busybox` may not be in path, use `self.busybox` ,e.g. https://github.com/ARM-software/devlib/blob/master/devlib/target.py#L209